packages/frontend-api/src/Model/Resolver/Parameter/ParameterResolverMap.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Resolver\Parameter;
+
+use Overblog\GraphQLBundle\Resolver\ResolverMap;
+use Shopsys\FrontendApiBundle\Model\Parameter\ParameterWithValues;
+
+class ParameterResolverMap extends ResolverMap
+{
+    /**
+     * @return array
+     */
+    protected function map(): array
+    {
+        return [
+            'Parameter' => [
+                'uuid' => function (ParameterWithValues $parameterWithValues) {
+                    return $parameterWithValues->getParameter()->getUuid();
+                },
+                'name' => function (ParameterWithValues $parameterWithValues) {
+                    return $parameterWithValues->getParameter()->getName();
+                },
+                'visible' => function (ParameterWithValues $parameterWithValues) {
+                    return $parameterWithValues->getParameter()->isVisible();
+                },
+            ],
+        ];
+    }
```

Comment:
Maybe this could be automated with some serializer inside `ParameterWithValues` which will make it more easier to extend.

But I really do not mind it now. It just an idea for future.

Reply:
done